types/console-log-level/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for console-log-level 1.4
+// Project: https://github.com/watson/console-log-level
+// Definitions by: Ali Ijaz Sheikh <https://github.com/ofrobots>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+declare enum LogLevels {
```

Comment:
It looks like this enum is only used for its keys. Why not just use a string union `"trace" | "debug" | ... | "fatal"`?